app/models/concerns/fog_extensions/kubevirt/server.rb

```diff
@@ -44,10 +44,10 @@ def reset
       end
 
       def vm_description
-        _("%{cpu_cores} Cores and %{memory} memory") % {:cpu_cores => cpu_cores, :memory => number_to_human_size(memory.to_i)}
+        format(_("%{cpu_cores} Cores and %{memory} memory"), :cpu_cores => cpu_cores, :memory => number_to_human_size(memory.to_i))
```

Comment:
Is format better than string interpolation?

Reply:
reverted

Reply:
I did not mean to revert it, I meant the "blabla #{expression here}" format. Anyway it is not very important.